### PR TITLE
Add Prerequisites pipeline2ATX step

### LINKED ISSUE #22: Add Prerequisites pipeline2ATX step
**Describe the feature request**

As a jenkins library maintainer I want to list all prerequisites within the method

**Acceptance Criteria**
* [ ] all prerequisites added

### PR DESCRIPTION
**Pull request by MarioFuchsTT**
Closes #22

### CODE REVIEW

**Review by MartinGroscheTT — APPROVED** [review]

File: vars/pipeline2ATX.groovy [hunk 1]
```diff
@@ -368,7 +372,12 @@ def createTestStep(row, debug) {
         description.addAll(crawlRows(child, true))
         child = child.nextTreeSibling
     }
-    testStep.put("description", description.join(""))
+    def allowedSchemaMaxStringLength = 117
```
Comment (MartinGroscheTT) [inline vars/pipeline2ATX.groovy:375]: allowedSchemaMaxStringLength  are 120 ;-)
Please add a comment why you use 117

File: vars/pipeline2ATX.groovy [hunk 2]
```diff
@@ -368,7 +372,12 @@ def createTestStep(row, debug) {
...
+    if(description.join("").isEmpty()) {
+        testStep.put("description", "")
+    } else {
+        testStep.put("description", description.join("").take(allowedSchemaMaxStringLength) + "...")
+    }
```
Comment (MartinGroscheTT) [inline vars/pipeline2ATX.groovy:380]: We don't care about the content of the pipeline prints.
```
if(description) {
    testStep.put("description", description.join(";").take(allowedSchemaMaxStringLength) + "...")
} else {
   testStep.put("description", "")
}
```
And we should use a better separator (e.g. ";")

**Review by MartinGroscheTT — CHANGES_REQUESTED** [review]

File: vars/pipeline2ATX.groovy (hunk 1 above)
Comment (MarioFuchsTT) [reply vars/pipeline2ATX.groovy]: resolved in [efd391f](https://github.com/tracetronic/jenkins-library/pull/23/commits/efd391f6ebccb315d0cfa6909cd4fbc588bb41d0)

File: vars/pipeline2ATX.groovy (hunk 2 above)
Comment (MarioFuchsTT) [reply vars/pipeline2ATX.groovy]: resolved in [efd391f](https://github.com/tracetronic/jenkins-library/pull/23/commits/efd391f6ebccb315d0cfa6909cd4fbc588bb41d0)

**Pull request merged**